tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Extend https://github.com/sindresorhus/tsconfig and remove the properties here that are already defined in that config.

Reply:
➕